File: Java/f(x)=ax+b.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        int b = input.nextInt();
        if (a == 0) 
            System.out.println("f(x)=" + b);
        else if (a == 1) 
        {
            if (b != 0)
                System.out.println("f(x)=x" + (b > 0 ? "+" : "") + b);
            else
                System.out.println("f(x)=x");
        } 
        else if (a == -1) 
        {
            if (b != 0)
                System.out.println("f(x)=-x" + (b > 0 ? "+" : "") + b);
            else
                System.out.println("f(x)=-x");
        } 
        else if (b == 0)
            System.out.println("f(x)=" + a + "x");
        else
            System.out.println("f(x)=" + a + "x" + (b > 0 ? "+" : "") + b);
        input.close();
    }
}

File: Python/f(x)=ax+b.py
a, b = map(int, input().split())
if a == 0:
    print(f"f(x)={b}")
elif a == 1:
    print(f"f(x)=x{'+' if b > 0 else ''}{b}" if b != 0 else "f(x)=x")
elif a == -1:
    print(f"f(x)=-x{'+' if b > 0 else ''}{b}" if b != 0 else "f(x)=-x")
elif b == 0:
    print(f"f(x)={a}x")
else:
    print(f"f(x)={a}x{'+' if b > 0 else ''}{b}")